### PR TITLE
add read data from database

### PR DESCRIPTION
**Pull request by xxwjj**
1.  add read data from database.

### CODE REVIEW

File: anomaly_detection/cmd/data_parser.py [hunk 1]
```diff
@@ -11,7 +11,27 @@
...
+from anomaly_detection.data_parser import manager
+
+CONF = cfg.CONF
 
 
 def main():
-    pass
+    CONF(sys.argv[1:])
+    log.setup(CONF, "anomaly_detection")
+    mgr = manager.Manager('csv')
```
Comment (xing-yang) [inline anomaly_detection/cmd/data_parser.py]: Why is this hard-coded to use 'csv'?  Should be from a config option.

File: anomaly_detection/data_parser/manager.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+    def run(self):
+        raise NotImplemented
+
+
+class CSVDataReceiver(DataReceiver):
+    def __init__(self):
+        super(CSVDataReceiver, self).__init__(name="csv")
+        self.once = False
+        self.csv_file = 'performance.csv'
```
Comment (xing-yang) [inline anomaly_detection/data_parser/manager.py]: File name should be configurable.

File: anomaly_detection/api/middleware/auth.py [hunk 3]
```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from werkzeug.wrappers.request import Request
+
```
Comment (xing-yang) [inline anomaly_detection/api/middleware/auth.py:15]: project_id should be changed to tenant_id

File: anomaly_detection/cmd/data_parser.py (hunk 1 above)
Comment (xxwjj) [reply anomaly_detection/cmd/data_parser.py]: done

File: anomaly_detection/data_parser/manager.py (hunk 2 above)
Comment (xxwjj) [reply anomaly_detection/data_parser/manager.py]: done

File: anomaly_detection/api/middleware/auth.py (hunk 3 above)
Comment (xxwjj) [reply anomaly_detection/api/middleware/auth.py:15]: done

**Review by xing-yang — APPROVED** [review]
lgtm

**Pull request merged**